content/en/docs/languages/go/generated-code.md

```diff
@@ -34,68 +35,29 @@ In this context, `MsgA` is the protobuf message sent from the client, and `MsgB`
 ### Server-streaming methods
 
 These methods have the following signature on the generated service interface:
+`Foo(*MsgA, grpc.ServerStreamingServer[*MsgB]) error`
```

Comment:
I think it would help to rename these from MsgA and MsgB to RequestMsg and ResponseMsg.  WDYT?

Reply:
Sounds good to me.

Reply:
Changed in the new doc, did not change in the documentation for old generated code for continuity of users who might have been referring to it. Or should it be changed in the old documentation too.